meeting_minutes/220224_SARIF_TC_53.md

```diff
@@ -0,0 +1,175 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company          | Role(s)                 |
+|:-----------|:----------|:-----------------|:------------------------|
+| Aditya     | Sharad    | GitHub           | Voting Member           |
+| Chris      | Meyer     | Microsoft        | Voting Member           |
+| David      | Keaton    | Individual       | Chair                   |
+| Eddy       | Nakamura  | Microsoft        | Voting member           |
+| Katrina    | O'Neil    | Micro Focus      | Voting member           |
+| Michael    | Fanning   | Microsoft        | Voting member           |
+| Nathan     | Baird     | Microsoft        | Voting Member           |
+| Paul       | Anderson  | Grammatech, Inc. | Voting member           |
+| Stefan     | Hagen     | Individual       | Secretary, taking notes |
+| Thanassis  | Avgerinos | ForAllSecure Inc | Member                  |
+
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/69645/agenda_20220224.html
+
+  Agenda was approved
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-02-10_Meeting #52](https://www.oasis-open.org/committees/document.php?document_id=69644&wg_abbrev=sarif)
+
+  Minutes were approved 
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION David: Find out how to upload content to wikipedia (#507)
+  * Ongoing - account granted
+* ACTION Michael: to set up meeting information so interested parties can participate on CIP meeting next week
+  * Completed
+* ACTION Michael: to open an issue in the repo for the numerics problem, tag it as technical topic to be solved by committee.
+  * Completed - issue is [Clarify integer (for properties such as location.id) size in spec #516](https://github.com/oasis-tcs/sarif-spec/issues/516)
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair Keaton)
+
+- Proposed Teleconferences (Thursdays at 08:00 PT / 16:00 UTC for 1.5 hours)
+    ```
+    March 10
+    ```
+- Proposed Teleconferences (Thursdays at 08:00 PDT / 15:00 UTC for 1.5 hours)
```

Comment:
"Proposed" should be "Scheduled"

Reply:
Done (and received the invitations). Thanks.